### PR TITLE
bpo-19225: Remove duplicated description for standard warning categories

### PR DESCRIPTION
**Pull request by cocoatomo**
see [bpo-19225](http://bugs.python.org/issue19225).

### CODE REVIEW

File: Doc/c-api/exceptions.rst [hunk 1]
```diff
@@ -292,15 +292,8 @@ an error value).
...
 
    Warning categories must be subclasses of :c:data:`Warning`; the default warning
-   category is :c:data:`RuntimeWarning`.  The standard Python warning categories are
-   available as global variables whose names are ``PyExc_`` followed by the Python
-   exception name. These have the type :c:type:`PyObject\*`; they are all class
-   objects. Their names are :c:data:`PyExc_Warning`, :c:data:`PyExc_UserWarning`,
-   :c:data:`PyExc_UnicodeWarning`, :c:data:`PyExc_DeprecationWarning`,
-   :c:data:`PyExc_SyntaxWarning`, :c:data:`PyExc_RuntimeWarning`, and
-   :c:data:`PyExc_FutureWarning`.  :c:data:`PyExc_Warning` is a subclass of
-   :c:data:`PyExc_Exception`; the other warning categories are subclasses of
```
Comment (serhiy-storchaka) [inline Doc/c-api/exceptions.rst:302]: Perhaps the last sentence should be kept here or added to the Standard Warning Categories section (if it already is not duplicated anywhere).

Comment (cocoatomo) [reply Doc/c-api/exceptions.rst:302]: The sentence at the bottom line seems to say the same meaning.

> This is a base class for other standard warning categories.

Comment (serhiy-storchaka) [reply Doc/c-api/exceptions.rst:302]: But is it documented that PyExc_Warning is a subclass of PyExc_Exception?

Comment (cocoatomo) [reply Doc/c-api/exceptions.rst:302]: No, it isn't. So, that sentence should be put somewhere around the paragraph or Standard Warning Categories section.
It seems suitable for that sentence to sit at the beginning of the paragraph, doesn't it?

Comment (serhiy-storchaka) [reply Doc/c-api/exceptions.rst:302]: LGTM.

Comment (cocoatomo) [reply Doc/c-api/exceptions.rst:302]: I have updated the PR.

**Comment by cocoatomo** [comment]
@serhiy-storchaka Is this PR ready to merge?

File: Doc/c-api/exceptions.rst [hunk 2]
```diff
@@ -291,16 +291,10 @@ an error value).
...
-   :c:data:`PyExc_Warning`.
+   :c:data:`PyExc_Warning` is a subclass of :c:data:`PyExc_Exception`;
+   warning categories must be subclasses of :c:data:`Warning`; the default warning
```
Comment (serhiy-storchaka) [inline Doc/c-api/exceptions.rst]: `:c:data:'PyExc_Warning'`, not `:c:data:'Warning'`.

I think that the fact that PyExc_Warning is a subclass of PyExc_Exception should be mentioned after saying that warning categories must be subclasses of PyExc_Warning.

File: Doc/c-api/exceptions.rst [hunk 3]
```diff
@@ -291,16 +291,10 @@ an error value).
...
+   category is :c:data:`RuntimeWarning`. Their names are enumerated at
```
Comment (serhiy-storchaka) [inline Doc/c-api/exceptions.rst]: `:c:data:'PyErr_RuntimeWarning'`, not `:c:data:'RuntimeWarning'`.

Comment (serhiy-storchaka) [inline Doc/c-api/exceptions.rst]: "Their names" -- that are not all names. Only names of the standard Python warning categories.

Comment (serhiy-storchaka) [reply Doc/c-api/exceptions.rst]: Maybe: "The standard Python warning categories are available as global variables whose names are enumerated at...".

**Comment by cocoatomo** [comment]
@serhiy-storchaka Thank you for the comments. I updated the PR.

**Review by serhiy-storchaka — APPROVED** [review]

**Review by berkerpeksag — APPROVED** [review]

**Comment by berkerpeksag** [comment]
Thanks! I will do the backporting stuff tomorrow.

**Comment by cocoatomo** [comment]
@berkerpeksag Would it bother you if I created backporting PRs?

**Comment by berkerpeksag** [comment]
@cocoatomo not at all, go for it! :)

**Pull request merged**